File: testlib_checker_test/wa/submission.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

rl.on("line", (input) => {
    const n = parseInt(input);
    let arr = input.split(" ").map(Number);

    console.log(-1);
    rl.close();
});

File: testlib_checker_test/wa/submission.dart
import 'dart:io';
import 'dart:collection';

void main() {
    int? n = int.tryParse(stdin.readLineSync() ?? '');

    // take list input
    List<int> arr =  stdin.readLineSync()?.split(' ').map((e) => int.parse(e)).toList() ?? [];

    print(-1);
}